resources/definitions/tronxy_x.def.json

```diff
@@ -0,0 +1,267 @@
+{
+    "name": "Tronxy Base Printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "author": "AdderMk2",
+        "manufacturer": "Tronxy",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+
+        "machine_extruder_trains": {
+            "0": "tronxy_base_extruder_0"
+        },
+
+        "has_materials": true,
+        "has_variants": true,
+        "has_machine_quality": true,
+        "variants_name": "Nozzle Size",
+
+        "preferred_variant_name": "0.4mm Nozzle",
+        "preferred_quality_type": "standard",
+        "preferred_material": "generic_pla",
+        "exclude_materials": [
```

Comment:
You are excluding a lot of materials that will not be shown anyway because the filament diameter does not match the filament diameter of the printers: `generic_*` and `ultimaker_*`

Reply:
Not sure if you watch Dragonball Z but... *This isnt even my final form*
I re-uploaded this as a draft so I could work between two PC's but still utilize the build check on the page. MOST of the files in this are being revamped and should be done this weekend. The large majority of those exclusions were done due to filament diameter. I was unaware that the program would ignore them due to that value. This and various other changes will be rolled into the next file drop. Thanks for the heads up!